project-fortis-interfaces/src/components/Admin/Streams/CreateStream.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Form from 'react-jsonschema-form';
+import { Card, CardTitle, CardText } from 'material-ui/Card';
+import Divider from 'material-ui/Divider';
+import StreamConstants from './StreamConstants';
+import { guid } from './../../../utils/Utils';
+import forOwn from '../../../../node_modules/lodash/forOwn';
+
+class CreateStream extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.save = this.save.bind(this);
+
+    this.state = {
+      dropdownValue: StreamConstants.defaultStreamMap.bing
+    };
+  }
+
+  save = data => {
+    const stream = data.formData.stream;
+    const addGuidToStreamIfNotExist = () => { if (!stream.streamId || stream.streamId.length === 0) stream.streamId = guid(); }
+    
+    const formatParamsForGraphqlSchema = () => {
+      const paramEntries = [];
+      forOwn(stream.params, (value, key) => {
+        paramEntries.push({ key, value: (typeof value === 'boolean' ? value.toString() : value) });
+      });
+      if (paramEntries.filter(entry => entry.key === 'watchlistFilteringEnabled').length === 0) paramEntries.push({ key: "watchlistFilteringEnabled", value: "false" });

```

Comment:
You can use lodash's [find](https://lodash.com/docs/4.17.5#find) to make this clearer. Here, the if condition checks if the element with a particular key is in the array, right? So we can do this:

```js
import find from 'lodash/find';

if (!find(paramEntries, {key: 'watchlistFilteringEnabled'})) {
  // ...
}
```